src/cim/components/common/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const AGENT_LOCATION_LABEL_KEY =
+  'agentclusterinstalls.extensions.hive.openshift.io/location';
+
+export const RESERVED_AGENT_LABEL_KEY =
```

Comment:
Consider reusing https://github.com/openshift/assisted-service/blob/master/api/v1beta1/agent_types.go#L168 .

Can we patch it before installation start?
Can a ClusterDeployment's finalizer clean it?